conftest.py

```diff
@@ -63,9 +64,16 @@ def pytest_addoption(parser):
         default=False,
         help="Ignore SSH banners when SSH commands are executed"
     )
+    parser.addoption(
+        "--ssh-debug-output-max-lines",
+        action="store",
+        default=20,
+        help="Max lines to output in a debug log (0 if no limit)"
+    )
 
 def pytest_configure(config):
     global_config.ignore_ssh_banner = config.getoption('--ignore-ssh-banner')
+    global_config.max_log_lines = int(config.getoption('--ssh-debug-output-max-lines'))
```

Comment:
I think it would be better to make the CLI option and the stored parameter to have the same name. `max_log_lines` is shorter and seems fine.

Reply:
the name would be confusing imho.

Reply:
Then isn't it confusing in config.py already?

Reply:
Probably ^^'
But I think we should find a better name which would be used for both

Reply:
`ssh-output-max-lines` is more concise since it has nothing to do with debu specifically

Reply:
Looks good